newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php

```diff
@@ -36,7 +36,7 @@ public function indexAction(Request $request)
         $blogService = $this->get('blog');
         $user = $userService->getCurrentUser();
 
-        if ($blogService->isBlogger($user)) {
```

Comment:
what if someone customized permissions and enable comments moderation for bloggers accidently ? If bloggers shouldnt have comments moderation rights at all there should be also a check if the user is blogger or not